test/groovy/PiperStageWrapperTest.groovy

```diff
@@ -181,6 +185,25 @@ class PiperStageWrapperTest extends BasePiperTest {
         assertThat(DebugReport.instance.localExtensions.test_old_extension, is('Extends'))
     }
 
+    @Test
+    void testExtensionExec() {
+        helper.registerAllowedMethod('fileExists', [String.class], { path ->
+            return (path == '.pipeline/extensions/test_old_extension.groovy')
+        })
+
+        nullScript.commonPipelineEnvironment.gitBranch = 'testBranch'
+        nullScript.env.PIPER_DISABLE_EXTENSIONS = true
```

Comment:
This sets to the Groovy boolean value `true`, but on OS level I think all env vars are `strings`.